app/components/RegionDropdown/RegionDropdown.module.scss

```diff
@@ -48,12 +48,13 @@ $control-width: 260px; // Control width local variable for now
 
 .dropdown-caret {
   position: absolute;
-  right: $space-xl;
-  top: $space-half;
+  right: $space-l;
+  top: calc($space-half);
```

Comment:
Why does this need to be a `calc`?

Reply:
Woops! Was playing around with the layout and forgot to revert this!